### PR TITLE
DO NOT MERGE - Make David happy for once

### PR DESCRIPTION
**Pull request by smarterclayton**
@deads2k - The last commit removes the kubelet from Origin.

However, we may have a problem.  How is volume quota being enforced in the emptydir plugin in standalone hyperkube?  Did we land the patch there?

### CODE REVIEW

**Comment by deads2k** [comment]
I love that the title clearly indicate that this will make me happy and you want to be sure that no one merges it!

File: pkg/cmd/openshift-controller-manager/controller/build.go [hunk 1]
```diff
@@ -57,7 +57,7 @@ func RunBuildController(ctx ControllerContext) (bool, error) {
 			Codec: annotationCodec,
 		},
 		SourceBuildStrategy: &buildstrategy.SourceBuildStrategy{
-			Image: imageTemplate.ExpandOrDie("sti-builder"),
+			Image: imageTemplate.ExpandOrDie("docker-builder"),
```
Comment (deads2k) [inline pkg/cmd/openshift-controller-manager/controller/build.go]: this surprises me.  Why would this image change?

**Comment by deads2k** [comment]
@smarterclayton any substantive golang changes or just a bunch of moves?

File: cmd/openshift-node-config/openshift-node-config.go [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+	defer logs.FlushLogs()
+
+	rand.Seed(time.Now().UTC().UnixNano())
+
+	var configFile string
+
+	cmd := &cobra.Command{
+		Use: "openshift-node-config",
+		Long: heredoc.Doc(`
+			Generate Kubelet configuration from node-config.yaml
```
Comment (mfojtik) [inline cmd/openshift-node-config/openshift-node-config.go]: nit: Generate Kubelet flags from node-config ?

File: pkg/oc/cli/cmd/export.go [hunk 3]
```diff
@@ -66,6 +66,8 @@ func NewCmdExport(fullName string, f *clientcmd.Factory, in io.Reader, out io.Wr
 			kcmdutil.CheckErr(err)
 		},
 	}
+	cmd.Deprecated = "Use the `get --export` command instead."
```
Comment (soltysh) [inline pkg/oc/cli/cmd/export.go]: :tada: :tada: :tada:

Comment (soltysh) [reply pkg/oc/cli/cmd/export.go]: Please set `Hidden` as well, I need to swipe through all our commands which are deprecated and set that flag, so they don't appear in completion, or in help.

**Pull request closed without merging**